app.py

```diff
@@ -181,6 +188,29 @@ def delete_patient(folder_hash):
 
     return redirect(url_for('dashboard'))
 
+@app.route('/add_data/add_biopsy/<folder_hash>', methods=['POST'])
```

Comment:
>add_data/add_biopsy [](start = 13, length = 19)

html appears to go to add_biopsy

Reply:
I forgot to make the add_biopsy page! it was going to patient info page. sorry.